src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -169,6 +182,7 @@
 -- retention_policy            RTP
 -- transcription               TRA
 -- transcription_comment       TRC
+-- trnascription_document      TRD

```

Comment:
Typo

Reply:
will be fixed in v52 of the application schema script